sessionctx/variable/tidb_vars.go

```diff
@@ -584,6 +584,10 @@ const (
 	TiDBEnableGlobalTemporaryTable = "tidb_enable_global_temporary_table"
 	// TiDBEnableLocalTxn indicates whether to enable Local Txn.
 	TiDBEnableLocalTxn = "tidb_enable_local_txn"
+	// TiDBTSOClientBatchMaxWaitTime indicates the max value of the TSO Batch Wait interval time of PD client.
+	TiDBTSOClientBatchMaxWaitTime = "tidb_tso_client_batch_max_wait_time"
+	// TiDBTSOEnableFollowerProxy indicates whether to enable the TSO Follower Proxy feature of PD client.
+	TiDBTSOEnableFollowerProxy = "tidb_tso_enable_follower_proxy"
```

Comment:
Our convention for other enable booleans is: `tidb_enable_XX`. Is it okay to change it?

Reply:
It's a discussion result with our PM Eason (He has no GitHub ID). We think using `tidb_tso ` prefix here to distinguish by different functional modules is better. Since these two new features are from the PD client rather than TiDB itself.

Reply:
There are no other tidb_tso_* variables though, and so far we are consistent with tidb_enable for booleans :(

I previously [worked on a PR](https://github.com/pingcap/tidb/pull/21988), and my reviewers (and PM) asked them to be named tidb_gc_* even though the names in the mysql.tidb table are tikv_gc_*

Reply:
See also: https://docs.google.com/document/d/1lGrfwdlv6ARcyAVrdXbQ5OYeFiVybBCcr172WCvAafI/edit?n=Sysvar_Usage_Proposal#heading=h.efxhx09hxyyo

Reply:
Sorry, we didn't know this context before, and since that's the case, I think we'd better keep it consistent with the previous naming convention.

Reply:
And as the flag naming convention, `tidb_enable` prefix is a higher level than the module name makes sense too. You're right about this. Thanks for pointing it out🤝.

Reply:
I use `SHOW VARIABLES LIKE 'tidb_enable%'` all the time. So thanks for making it consistent :-)